manager/base.py

```diff
@@ -66,6 +66,20 @@ class reporter(Enum):  # pylint: disable=invalid-name
     BOTH = 3
 
 
+class ServiceAccess:
+    """Stores access status to external service"""
+    def __init__(self, status: bool):
+        self._status: bool = status
+
+    def get_status(self) -> bool:
```

Comment:
getters and setters are useless in python, the variable could be accessed or changed by `patch_access.status = True`